.github/CONTRIBUTING.md

```diff
@@ -10,11 +10,12 @@
 ## How to run tests
 
 Since most of the tests are actually integration tests you will need to have a
-working docker setup to make the full test suite pass.
+working docker setup to make the full test suite pass. gtest build requires python 2 :(
 
 1. build vendorized gtest: `./utils/build_gtest.sh`
-2. build restclient-cpp: `./autogen.sh && ./configure && make check`
-3. run the unit test suite: `make ci`
+1. build restclient-cpp: `./autogen.sh && ./configure && make check`
```

Comment:
List numbering accidentally lost?

Reply:
It's intentional. Markdown syntax allows you to not have to explicitly number lists, so that when you add a number in the middle, you don't have to update all the numbers below it (like in this case).

Reply:
Wow. Thanks. I didn't know about this Markdown syntax feature.